packages/react-components/src/components/AttachmentCard.tsx

```diff
@@ -64,9 +60,9 @@ export interface _AttachmentCardProps {
  * `_AttachmentCard` internally uses the `Card` component from `@fluentui/react-components`. You can checkout the details about these components [here](https://react.fluentui.dev/?path=/docs/components-card).
  */
 export const _AttachmentCard = (props: _AttachmentCardProps): JSX.Element => {
-  const { attachment, progress, menuActions, onActionHandlerFailed } = props;
+  const { attachment, menuActions, onActionHandlerFailed } = props;
   const attachmentCardStyles = useAttachmentCardStyles();
-
+  const progress = attachment.progress;
```

Comment:
Please use `useMemo` here as otherwise progress will trigger rendering of com-onent where it's used even when it's not changed